### DISCUSSION TITLE
Is there any particular reason why the GitHub Education category isn't split up?

### DISCUSSION THREAD

**Discussion by suprstarrd** [community/community#17781] [GitHub Education]
> Ask questions about GitHub Education products. Campus Experts, Global Campus, GitHub Classroom, etc.

This feels like a perfect way to demonstrate how awesome Discussions is and split these up into separate categories like [github/feedback](https://github.com/github/feedback). Having them all in one category feels very strange. Is there any particular reason why these aren't split up?

**Comment by ettaboyle** [comment]
Thanks for the feedback @jbmagination 🤔  The github/feedback discussion is really active, and has been a great place for the teams here to learn how GitHub users are using Discussions. 

What categories do you think would be helpful to better organize topics around education?

**Reply by suprstarrd** [reply]
I think topics around education is important, but mainly people are here in this category for the *GitHub Education* platform. Referencing `education.github.community`, we had these discussion groups:
- GitHub Campus Program
- Teachers
- GitHub Classroom
- Students
- Miscellaneous
- Meta

From what I can tell (in my likely poor attempts at understanding Discourse) Meta was the least used topic, and for good reason. Announcements already fulfills that role for the most part. Miscellaneous was largely underused but Students was where most of that went.

The Education category on `github.community` additionally referenced GitHub Learning Lab.

GitHub Campus Program was largely used for Campus Experts, while the Student Pack and Teacher Toolbox had most discussion in the Students and Teachers category (although there was discussion in the Campus Program category, too!). I can see GitHub Classroom was a very used category, although I have a feeling it could do better in github/feedback.

With all of this in mind, at the moment my idea of splitting things up would be something like this:
- Global Campus Teachers
- Global Campus Students
- GitHub Classroom
- GitHub Learning Lab
- Show-and-tell
- General Education

**Reply by ettaboyle** [reply]
Thanks for giving the time and attention to this! I'll make sure the rest of the team sees this as well! 😄
This is the primary discussion in the GitHub Community Org, but the org is also home to private repos that are dedicated to programs where users have provisioned access (ex: Global Campus Teachers). Repo & Org level Discussions are still really new so I know the team is being very thoughtful with how we roll out new categories, and how we use tags, to make sure content is easy to find. Hearing feedback from users (especially folks who jumped right in like you!) is incredibly helpful! Thank you! 💛

**Comment by suprstarrd** [comment]
In reference to https://github.com/github-community/community/discussions/29#discussioncomment-2640951: I should probably add that I know Announcements (and Polls?) probably can't be removed; that's totally fine. It's also fine if any category is added/removed, or if this just doesn't happen at all. I do think that splitting it up would've made this migration more familiar for those used to the Discourse board.
